### PR TITLE
Add setting locale 

### PR DESCRIPTION
**Pull request by PlexPt**

### CODE REVIEW

File: settings.lua [hunk 1]
```diff
@@ -9,15 +9,16 @@ local settings =
...
     maximum_value = 80085
   },
-  
+
   {
     type = "string-setting",
     name = "fuel-fluid",
     localised_name = "Transport drone fuel",
     setting_type = "startup",
+    allowed_values = {"petroleum-gas", "water", "crude-oil", "heavy-oil", "light-oil", "lubricant", "steam", "sulfuric-acid"},
```
Comment (Klonan) [inline settings.lua:18]: IDK, people play with mods, and specifically this option is here for weird mods with other fuels, so limiting the allowed values defeats the utility

Comment (PlexPt) [reply settings.lua:18]: But most people(99.999999%) only need petroleum-gas

**Comment by PlexPt** [comment]
Is this ok?

File: settings.lua (hunk 1 above)
Comment (Klonan) [reply settings.lua:18]: The setting is there for the 1% of cases, and also with some mods they remove petroleum gas or something, so the setting should be able to be set to any valid fluid

**Comment by PlexPt** [comment]
I removed the restriction

File: settings.lua (hunk 1 above)
Comment (PlexPt) [reply settings.lua]: https://github.com/Klonan/Transport_Drones/pull/32/commits/d3daa1b1a7ffb57d06bbb324f154e3b0f7e78f0d

Comment (PlexPt) [reply settings.lua]: @Klonan

**Pull request merged**